dplutils/observer/ray.py

```diff
@@ -17,15 +17,20 @@ class RayActorWrappedObserver(Observer):
     """
     def __init__(self, cls, *args, **kwargs):
         self.actor = ray.remote(cls).remote(*args, **kwargs)
+        self._wait = False  # for testing purposes. If true wait instead of fire-and-forget
+
+    def _wrap_waiter(self, remote):
+        if self._wait:
+            ray.get(remote)
```

Comment:
Is there an error handling strategy for ray.get(remote) in _wrap_waiter?

Reply:
This is really only for testing so I think it is not important here, otherwise if we'd want to do synchronous fire we'd probably just ignore the exception - and if not then it would trickle up anyhow so I don't think anything needs to be done here

Reply:
understood. It's reasonable to keep things simple for testing purpose